### PR TITLE
Translate some ML terms to Spanish

### PR DESCRIPTION
**Pull request by paocorrales**
## Author: 

- Pao Corrales 

## Language: 

- Spanish

## Terms defined:

- Hidden layer
- ROC curve

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -10442,6 +10442,12 @@
       A hidden layer in a [neural network](#neural_network) refers to the layers of neurons that are
       not directly connected to input or output. The layers are "hidden" because you do not directly
       observe their input and output values.
+  es:
+    term: capa oculta (deep learning)
+    def: >
+      Una capa oculta en una [red neuronal](#neural_network) hace referencia a las capas de neuronas que
+      no están directamente conectadas al ingreso o salida de información. Las capas están "escondidas" porque
+      no puedes observadirectamente el ingreso y salida de valores.
```
Comment (nicoguaro) [inline glossary.yml:10450]: There is an r and space missing here

File: glossary.yml [hunk 2]
```diff
@@ -10454,4 +10460,11 @@
       of a binary classifier at different [classification](#classification) thresholds. The curve is 
       obtained by plotting the True Positive Rate (also known as Recall or [Sensitivity](#sensitivity))
       along the vertical axis and the False Positive Rate along the horizontal axis.
+  es:
+    term: "Curva ROC"
+    def: >
+      Una curva ROC (acrónimo de Receiver Operating Characteristic, o Característica Operativa del Receptor) 
+      es un gráfico que muestra el desempeño de un clasificador binario con diferentes umbrales de [classificación](#classification)
```
Comment (nicoguaro) [inline glossary.yml:10467]: "classificación" should be "clasificación"

"Sensitividad" should be "Sensibilidad"

"taza" should be "tasa"

**Review by nicoguaro — CHANGES_REQUESTED** [review]
There are some minor mistakes in the translation

File: glossary.yml (hunk 1 above)
Comment (paocorrales) [inline glossary.yml:10450]:
```suggestion
      no puedes observar directamente el ingreso y salida de valores.
```

File: glossary.yml (hunk 2 above)
Comment (paocorrales) [inline glossary.yml:10467]:
```suggestion
      es un gráfico que muestra el desempeño de un clasificador binario con diferentes umbrales de [clasificación](#classification)
```

File: glossary.yml [hunk 3]
```diff
@@ -10454,4 +10460,11 @@
...
+      La curva se obtiene graficando la taza de verdaderos positivos (también conocida como Recall o [Sensitividad](#sensitivity))
```
Comment (paocorrales) [inline glossary.yml:10468]:
```suggestion
      La curva se obtiene graficando la tasa de verdaderos positivos (también conocida como Recall o [Sensibilidad](#sensitivity))
```

**Comment by elletjies** [comment]
@nicoguaro  Could you check the changes to the translation again please?

**Review by nicoguaro — APPROVED** [review]

**Comment by elletjies** [comment]
@nicoguaro thank you for the review!

**Comment by elletjies** [comment]
@all-contributors please add @paocorrales for translation

**Pull request merged**